STM32Resources/install.sh

```diff
@@ -6,7 +6,19 @@ if [[ $UID != 0 ]]; then
     exit 1
 fi
 
-apt-get -y install build-essential git libsane:i386 ia32-libs-multiarch autoconf libusb-1.0-0-dev lib32ncurses5 libncurses5:i386 software-properties-common pkg-config cmake
+apt-get update
```

Comment:
Excellent fix, can you put this into a separate PR so we can merge it first?

Reply:
What does this do?

Reply:
This command updates the source locations for all the packages, so updates where to download them and stuff, kind of. So before running apt-get install commands it's usually a good idea to include it or users can get weird errors, especially on fresh installs.